client/src/components/login/ProfilePage.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+import axios from "axios";
+
+const ProfilePage = () => {
+  const { user, isAuthenticated } = useAuth0();
+
+    const storeUserInfo = async () => {
+      try {
+        if (user) {
+          const userData = {
+            name: user.given_name || user.name || "DefaultName",
+            email: user.email || "DefaultEmail@example.com",
+            picture: user.picture || "default-profile-picture-url",
+            sub: user.sub,
+          };
+
+          const response = await axios.post(
+            "http://localhost:5005/storeUserInfo",
+            userData,
+            {
+              headers: {
+                "Content-Type": "application/json",
+              },
+            }
+          );
+
+          if (response.status < 200 || response.status >= 300) {
+            throw new Error(`HTTP error! Status: ${response.status}`);
+          }
+
+          const data = response.data;
+          console.log(data);
+        } else {
+          console.log("Error storing user: 'user' is undefined");
+        }
+      } catch (error) {
+        console.log("Error storing user:", error);
+      }
+    };
+
+    storeUserInfo();
```

Comment:
Question: why this page in particular stores the user info?

Reply:
is just showing the user profile as page

Reply:
oh.. I got what you mean .. and I move this function to login component.

the answer for your Q No .. because I have check condition in the endpoint checking if this user existingUser or not